rnote-compose/src/style/rough/mod.rs

```diff
@@ -63,6 +66,55 @@ impl Composer<RoughOptions> for Line {
     }
 }
 
+impl Composer<RoughOptions> for Arrow {
+    fn composed_bounds(&self, options: &RoughOptions) -> p2d::bounding_volume::Aabb {
+        let rline = self.compute_rline(Some(options.stroke_width));
+        let lline = self.compute_lline(Some(options.stroke_width));
+
+        let mut bounds = self.bounds();
+        bounds.take_point(rline.into());
+        bounds.take_point(lline.into());
```

Comment:
we can avoid having to compute the tip lines twice here and in `bounds()` by also using `Aaab::from_points()` here

Reply:
Ok, I updated this part of the code on my local branch but I somehow can't build rnote anymore:
```
(rnote:86155): GLib-GIO-ERROR **: 21:48:25.180: Settings schema 'com.github.flxzt.rnote.Devel' does not contain a key named 'block-pinch-zoom'
[1]    86155 trace trap (core dumped)  ./target/bin/rnote
```

I'll try to build it again after applying all the suggested changes.

Reply:
How are you building Rnote? If you use the Flatpak VS Code extension, this shouldn't happen. If you're using meson, you'll also have to install the build (`meson install -C _mesonbuild`) because the schema changed. Alternatively, I *think* that you can manually specify the schema path.

Reply:
I'm doing it as follows:
```bash
cd /home/tornax/Programming/pull_requests/rnote
meson setup --prefix=/home/tornax/Programming/pull_requests/rnote/target -Dprofile=devel build
meson compile -C build
meson install -C build
./target/bin/rnote
```

Reply:
and if I just want to rebuild everything:
```bash
meson compile -C build
meson install -C build
./target/bin/rnote
```